utils/filesystem/files.go

```diff
@@ -1165,6 +1159,34 @@ func (fs *VFS) copyZipFile(dest string, zippedFile *zip.File) (err error) {
 	return
 }
 
+func determineUnzippedFilepath(dest string) (string, error) {
+	destinationPath := dest
+
+	// Currently an error is raised when non-UTF8 characters are present in filepath
+	// See https://go-review.googlesource.com/c/go/+/75592/
+	// Character encodings other than CP-437 and UTF-8
+	//are not officially supported by the ZIP specification, pragmatically
+	//the world has permitted use of them.
+	//
+	//When a non-standard encoding is used, it is the user's responsibility
+	//to ensure that the target system is expecting the encoding used
+	//(e.g., producing a ZIP file you know is used on a Chinese version of Windows).
+	if !utf8.ValidString(destinationPath) {
```

Comment:
This is a preference thing rather than a requirement but I would write this the other way round.

```
if utf8.ValidString(destinationPath) {
    return desinationPath, nil
}
```

Just to avoid a function where most of it is indented in the if.

Reply:
this is a fair point. I changed the function